.travis.yml

```diff
@@ -3,24 +3,19 @@ language: python
 cache: pip
 env: TRAVIS=1
 
+install:
+    - pip install cython
+    - pip install -r requirements.txt -r requirements-dev.txt -e .
+
```

Comment:
I'm not sure if I feel like blocking PR from merging because of code style yet. The problem is that only @moonso can force merge and if he's not available in that moment then we will be blocked as before. What do the others think @moonso and @dnil ?  If you all agree then it's fine. I'd like to use black as a way of improvement my coding (I'd like my PRs to pass the formatting and the lynting) but I personally don't want become a slave of the system. Tests should be helping us improving and not slow us down and limit us in our work..

Reply:
I agree, lets live this a while without enforcement. I promise to have a look at what it says - and trust me, I Do Not Like extra diff lines with just whitespace, commas, quotation changes or other redundant code formatting, so you will not get much in the way of bystander changes out of style from me. Unless the black code is ugly ofcourse. :wink: